src/partials/setupInstructions.html

```diff
@@ -16,100 +23,196 @@ <h2 class="title is-2 light-text is-spaced pb-40">Setup instructions</h2>
+                            <li class="codeContainer">
+                                <span class="step">4.</span>
+                                <div class="code">
+                                    ./vedran-daemon --id=<span class="hightlight">&#60;some secret id for your
+                                        node&#62;</span> --lb=<span class="hightlight">&#60;load balancer
+                                        url&#62;</span>
+                                    --payout-address=<span class="hightlight">&#60;polkadot account address&#62;</span>
+                                </div>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
             </div>
         </div>
         <div id="linuxContainer" class="content-tab" style="display: none">
             <div class="box">
-                <h1>Linux</h1>
-                <h3 class="subtitle is-5">Vedran  Load Balancer</h3>
-                <ul>    
-                    <li><span class="step">1.</span> Download latest release from: <a target="_blank" href="https://github.com/NodeFactoryIo/vedran/releases" >Github</a></li>
-                    <li><span class="step">2.</span>Create polkadot account - <a target="_blank" href="https://wiki.polkadot.network/docs/en/learn-account-generation">wiki.polkadot.network</a></li>
-                    <li class="codeContainer">
-                        <span class="step">3.</span>
-                        <div class="code">
-                            ./vedran start --auth-secret=<span class="hightlight">&#60;generate random secret&#62;</span>   --private-key=<span class="hightlight">&#60;private seed from generated account&#62;</span>
-                        </div> 
-                    </li>
-                    <li><span class="step">4.</span> Use <span class="code">--help</span> for additional configuration and flags</li>
-                </ul>
-                <h3 class="subtitle is-5">Vedran Daemon</h3>
-                <ul>
-                    <li><span class="step">1.</span> Download latest release: <a href="https://github.com/NodeFactoryIo/vedran-daemon/releases">Github</a></li>
-                    <li class="codeContainer">
-                        <span class="step">2.</span>
-                        <div class="code">
-                                ./vedran-daemon --id=<span class="hightlight">&#60;some secret id for your node&#62;</span> --lb=<span class="hightlight">&#60;load balancer url&#62;</span>
-                                --payout-address=<span class="hightlight">&#60;polkadot account address&#62;</span>
+                <div class="columns  pb-20">
+                    <div class="tile column is-half is-vertical">
+                        <h3 class="subtitle is-4">Vedran Load Balancer</h3>
+                        <div class="pb-20" style="margin-left: 0.5rem;">
+                            <span class="box is-size-6" style="width: 70%;">
```

Comment:
Can we reuse this description as a component? same typos here :D

Reply:
we could but it seems like too much effort 😅